kbase-extension/static/kbase/js/common/monoBus.js

```diff
@@ -605,14 +571,6 @@ define(['uuid', 'bluebird', './lang', './unodep'], (Uuid, Promise, lang, utils)
             });
         }
 
-        function logMessages(doLog) {
-            if (doLog) {
-                doLogMessages = true;
-            } else {
-                doLogMessages = false;
-            }
-        }
```

Comment:
this does nothing as the variable `doLogMessages` isn't used anywhere. I also deleted references to this function elsewhere in the codebase